src/service/localization.js

```diff
@@ -90,10 +91,16 @@ export function getLanguageCodesFromString(languageCode) {
 export class LocalizationService {
   /**
    * @param {!Window} win
+   * @param {!./viewer-interface.ViewerInterface} viewer
    */
-  constructor(win) {
+  constructor(win, viewer = undefined) {
     const rootEl = win.document.documentElement;
 
+    /**
+     * @private {?string}
+     */
+    this.viewerLanguageCode_ = viewer ? viewer.getParam('lang') : null;
```

Comment:
Nit: the viewer service is always present, even when there's no actual viewer

Reply:
True, but it's not necessarily passed to this constructor (or, are you suggesting that we should make it a required argument?)

Reply:
Got it, sorry, I misread this PR